packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -81,7 +91,7 @@ export interface TreeNodeSchemaClass<
 	 * Constructs an {@link Unhydrated} node with this schema.
 	 * @remarks
 	 * This constructor is also used internally to construct hydrated nodes with a different parameter type.
-	 * Therefor overriding this constructor is not type-safe and is not supported.
+	 * Therefor overriding this constructor with different argument types is not type-safe and is not supported.
```

Comment:
```suggestion
	 * Therefore, overriding this constructor with different argument types is not type-safe and is not supported.
```
My understanding is that "Therefor" is used to mean "for that" and is typically used at the end of a sentence